backend/app/views/spree/admin/reports/_filters.html.erb

```diff
@@ -36,11 +42,16 @@
             <i class="icon icon-chevron-left"></i>
           </label>
         </div>
-        <select class="custom-select js-filter-node" id="reports-group-by" data-param="group_by">
-          <option selected value="day">Day</option>
-          <option value="month">Month</option>
-          <option value="year">Year</option>
-        </select>
+        <%= select_tag "reports-group-by",
+          options_for_select(
+            [[ "Day", "day" ],
```

Comment:
@devilcoders and here

Reply:
@damianlegawiec This will be changed in final version of PR